src/compiler/code_generator.ts

```diff
@@ -1162,4 +1193,17 @@ export class CodeGenerator {
       this.compileAST(ast.content, Object.assign({}, ctx, { translate: false }));
     }
   }
+  compileTPortal(ast: ASTTPortal, ctx: Context) {
+    this.helpers.add("callPortal");
+    let { block } = ctx;
+    if (ast.content) {
```

Comment:
hmm, I think this if statement is not necessary. There is always a ast.content value, no?